body-pix/demos/index.js

```diff
@@ -98,6 +99,14 @@ function getFacingMode(cameraLabel) {
   }
 }
 
+function getBackendFromQueryString() {
+  if (!window.URLSearchParams) return 'gpu';
```

Comment:
define CPU_STRING='cpu', and GPU_STRING='gpu' to avoid duplicate strings

Reply:
Done